packages/jupyter-ai/jupyter_ai/handlers.py

```diff
@@ -127,8 +127,8 @@ def get_chat_user(self) -> ChatUser:
             chat_user_kwargs = {
                 # set in case IdentityProvider doesn't return initials, e.g.
                 # JupyterHub (#302)
-                "initials": initials,
                 **asdict(self.current_user),
+                "initials": initials,
```

Comment:
Also, can you move the comment up to the definition of `default_initials` while you're at it?

Reply:
Thanks @dlqqq, moved the comments and set up the `default_initials`.

Unfortunately, it seems like the `getattr()` doesn't actually give the default value of `default_initials` if the `initials` attr exists.
This actually sets the `initials` key value back to `None` because `self.current_user.initials` technically is assigned a value of None.

I added the latest commit to address this and cover all cases to make sure `initials` is assigned: https://github.com/jupyterlab/jupyter-ai/pull/443/commits/ec00e2b6f8765d58a4b161043fa8a9094be89417